src/routes/students/Profile.svelte

```diff
@@ -1 +1,72 @@
+<script>
+  import {getProfile, getEmails, addEmail, setPrimaryEmail, deleteEmail} from '../../api'
+
+  const profilePromise = getProfile();
+
+  const emailsPromise = getEmails();
+
+  const addEmailButton = () => {
+    const newEmail = document.getElementById('addEmailField').value;
+    addEmail(newEmail);
+  };
+
+  const setPrimaryEmailButton = (emailId) => {
+      setPrimaryEmail(emailId).then(res => {
+        console.log(res);
+        getEmails();
+      });
+  };
+
+  const removeEmail = (emailId) => {
+      deleteEmail(emailId).then(res => {
+        console.log(res);
+      });
+  }
+</script>
+
+<style>
+    .email{
+        border: black solid 1px;
+        margin-bottom: 10px;
+    }
+</style>
+
 <div>Profile</div>
+
+<div>
+  {#await profilePromise}
+    <p>...loading profile</p>
+  {:then profile}
+    {profile['first_name']}<br>
+    {profile['last_name']}<br>
+    {profile['email']}<br>
+
+    <label>
+      <input id="addEmailField"/>
+    </label>
+    <button on:click={addEmailButton}>add email</button>
```

Comment:
On failure: show validation error (or at least something so the user know the action was not successful)

On success: clear input field

Reply:
Still does not work for me and I can't find any changes in your latests commits that touched this, do you not have this issue?

On failure: show validation error (or at least something so the user know the action was not successful)

On success: clear input field

Reply:
Added a short text field where it will show the error message on an unsuccessful add email attempt.

A successful add email attempt will clear the email field and any error messages that were there.

Reply:
Error message is not visible `Uncaught (in promise) TypeError: Cannot read property '0' of undefined
    at Profile.svelte:15`

Reply:
Added more (basic) error handling in frontend as bandaid. We should standardize error propagation in the backend to avoid the issue altogether.